sphinx/domains/std.py

```diff
@@ -406,15 +406,17 @@ def run(self) -> List[Node]:
         return messages + [node]
 
 
-def token_xrefs(text: str) -> List[Node]:
+def token_xrefs(text: str, productionGroup: str) -> List[Node]:
+    if len(productionGroup) != 0:
+        productionGroup += ':'
```

Comment:
Which is better `tokenGroup` or `productionGroup`?

Reply:
Good point, changed to be *productionGroup* in the documentation.